Standardize cloud resource references (#1700)
astronomer/docs: astro/resource-reference-gcp.md

| [Google Cloud Storage (GCS) Bucket](https://cloud.google.com/storage/docs/creating-buckets)          | Stores Airflow task logs.                                                                                                                                                                                                                                    | 1 bucket with name `airflow-logs-<clusterid>`                                              |
| Maximum Node Count                                                                                   | The maximum number of worker nodes that your Astro cluster can support. When this limit is reached, your Astro cluster can't auto-scale and worker Pods may fail to schedule.                                                                                             | 20                                                                                         | ✔️             |

## Supported cluster configurations

You might need to modify configurations of a new or existing cluster on Astro. This section provides a reference for cluster configuration options.

> To create a new cluster on Astro with a specified configuration, see [Create a cluster](create-cluster.md). To request a change to an existing Cluster, see [Modify a cluster](modify-cluster.md). Astronomer is currently responsible for completing all cluster configuration changes.

Reviewer: ```suggestion
To create a new cluster on Astro with a specified configuration, see [Create a cluster](create-cluster.md). To request a change to an existing cluster, see [Modify a cluster](modify-cluster.md). Astronomer is currently responsible for completing all cluster configuration changes.
```

### Worker node pools

A node pool is a group of nodes within a cluster that all have the same configuration. On Astro, worker nodes are responsible for running the Pods that execute Airflow tasks. Each worker node pool can be configured with a node instance type and a maximum node count. All Astro clusters have one worker node pool by default, but you can configure additional node pools to optimize resource usage.

If your cluster has multiple worker node pools with different worker node instance types, users in your organization can configure tasks to run on those worker types using [worker queues](configure-deployment-resources.md#worker-queues). To enable a new worker type for your cluster, contact [Astronomer support](https://cloud.astronomer.io/support) with a request to create a new node pool or modify an existing node pool.

> Astronomer monitors your usage and the number of nodes deployed in your cluster. As your usage of Airflow increases, Astronomer support might contact you and provide recommendations for updating your node pools to optimize your infrastructure spend or increase the efficiency of your tasks.

Reviewer: ```suggestion
Astronomer monitors your usage and the number of nodes deployed in your cluster. When your Airflow use increases, Astronomer support might contact you and provide recommendations for updating your node pools to optimize your infrastructure costs or increase the efficiency of your tasks.
```
